src/lib/functions/server.js

```diff
@@ -37,13 +37,23 @@ const buildClientContext = function (headers) {
   }
 }
 
+// same as https://github.com/netlify/bitballoon/blob/fbd7881e6c8e8c48e7a0145da4ee26090c794108/app/models/deploy.rb#L482
+// eslint-disable-next-line unicorn/better-regex
```

Comment:
Why do we need to disable this rule?

Reply:
b/c I'd like to keep the syntax exactly as in BitBalloon, to make it obvious that it's the same.